README.md

```diff
@@ -15,6 +15,9 @@ This extension adds language support for [Lingua Franca (LF)](https://www.lf-lan
 * target code validation upon build or file save
 * user-triggered build (<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd>, then `Lingua Franca: Build`) 
 * user-triggered build and run (<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd>, then `Lingua Franca: Build and Run`) 
+* Lingua Franca Package Explorer (click on the LF icon in the Activity Bar on the left), which features two distinct tree-view sections:
+   * **Local Libraries:** This section showcases a catalog of Local Libraries, which are libraries personally defined by the programmer.
```

Comment:
```suggestion
   * **Local Libraries:** This section showcases a catalog of Local Libraries, which are libraries personally defined by the programmer.
```
What does "personally" mean? And how does are libraries identified?

Reply:
I think he doesn't mean libraries (in the lingo sense) but individual reactors under `./src/lib`. 

The VSCode Extension indexes all the reactors under `./src/lib` and displays them.